src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(../src/fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: "Avenir";
+  font-weight: 700;
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+}
+
+/* Search bar */
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search-input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: "Avenir", Helvetica, sans-serif;
+  font-weight: 700;
+  background: url(../src/images/Search.svg) no-repeat;
+  outline: none;
+}
+
+.search-input::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-input:hover {
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-input:focus {
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+/* Big search bar */
+
+.big-search-form {
+  height: 70px;
+}
+
+.big-search-input {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  background-position: 25px 24px;
+  background-size: 19px;
+}
+
+.big-search-input:focus {
+  background: url(../src/images/Search.svg) no-repeat 25px 24px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+/* Small search bar */
+
+.small-search-form {
+  height: 42px;
+}
+
+.small-search-input {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+  background-position: 12px 14px;
+  background-size: 11px;
+}
+
+.small-search-input:focus {
+  background: url(../src/images/Search.svg) no-repeat 12px 14px/11px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
if there are many long rules, it is better to use separate rules for styling instead of background shorthand